crt_portal/cts_forms/filters.py

```diff
@@ -310,28 +336,36 @@ class OctetLength(Func):
     output_field = IntegerField()
 
 
-def filter_by_similar(queryset, column, search_text):
+def filter_by_similar(queryset, column, *, query, soundslike, lookslike):
```

Comment:
I'm a little confused about how the "sounds like" and "looks like" variables impact each other. Like if I set the looks like filter to be too strict will I exclude some of the sounds like filtering? We should probably explain that a bit for the user on the front end too.

Reply:
they OR together, so they don't actually impact each other - if you add more sounds like, you get more sounds like. If you then adds more looks like, you get more looks like in the mix, too.

Reply:
I've updated the tooltips, plus the chips should help express this now, too